pypdf/_writer.py

```diff
@@ -1902,44 +1918,10 @@ def remove_text(self, ignore_byte_string_object: bool = False) -> None:
         Remove text from this output.
 
         Args:
-            ignore_byte_string_object: optional parameter
+            ignore_byte_string_object: obsolete
```

Comment:
```suggestion
            ignore_byte_string_object: deprecated
```

Same as the other one :-)